cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_data_adapters.go

```diff
@@ -163,7 +163,7 @@ func (s *Server) buildInstalledPackageSummary(pkgInstall *packagingv1alpha1.Pack
 		IconUrl: iconStringBuilder.String(),
 		InstalledPackageRef: &corev1.InstalledPackageReference{
 			Context: &corev1.Context{
-				Namespace: pkgMetadata.Namespace,
+				Namespace: pkgInstall.Namespace,
```

Comment:
Previously the package metadata was requested in the same namespace (individually) as the package install, and the aggregate API (remember, not a CR), returns a result with that same namespace, so they were equivalent. Now that we're fetching all package metadatas once (potentially across all namespaces if requested by the user), the pkgMetadata.Namespace might be the global carvel one, rather than the one we're after for the installed package (ie. they can differ now).